database/seeds/OtherDataSeeder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+use Proto\Models\HashMapItem;
+use Proto\Models\User;
+use Proto\Models\Committee;
+use Proto\Models\Address;
+use Proto\Models\Bank;
+use Proto\Models\Product;
+use Proto\Models\Member;
+use Proto\Models\Activity;
+use Proto\Models\OrderLine;
+use Proto\Models\CommitteeMembership;
+use Proto\Models\ActivityParticipation;
+
+class OtherDataSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        $faker = Faker\Factory::create();
+
+        // List of valid ID's
+        $products = Product::all()->pluck('id')->toArray();
+        $committees = Committee::all()->pluck('id')->toArray();
+
+        // Create users
+        $n = 500;
+        echo "Creating $n users" . PHP_EOL;
+        factory(User::class, $n)->create()->each(function ($user) {
+            /** @var $user User */
+            if (mt_rand(1, 5) > 1) {
+                $user->address()->save(factory(Address::class)->make());
+            }
+            if (mt_rand(1, 5) > 1) {
+                $user->bank()->save(factory(Bank::class)->make());
+            }
+            if (mt_rand(1, 5) > 1) {
+                $user->member()->save(factory(Member::class)->make());
```

Comment:
Members required an address and bank details, so a check for presence of these is necessary. Alternatively, check if a user is member, then add address and bank, otherwise, add address and bank only at random.

Reply:
I now create address and bank at the same time as the member object, and sprinkle in some additional bank details and addresses for other users.